plugins/hadoop-authorization-sentry-default/hadoop-authorization-sentry/pom.xml

```diff
@@ -1,206 +1,219 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <project xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
-         xmlns="http://maven.apache.org/POM/4.0.0"
```

Comment:
Please ensure you're using the correct code formatter. XML files should be indented 2 spaces instead of 8.
http://kylo.readthedocs.io/en/latest/developer-guides/ContributingtoKylo.html#development-guidelines